system-tests/lib/fixtures.ts

```diff
@@ -11,6 +11,21 @@ const serverRoot = _path.join(__dirname, '../../packages/server/')
 const projects = _path.join(root, 'projects')
 const cyTmpDir = _path.join(tempDir, 'cy-projects')
 
+const safeRemove = (path) => {
```

Comment:
Could this leave some files in the file system after running all tests if it happens at the wrong moment?
Maybe we could do a complete cleanup at the end of the test cycle with a short delay to make sure we have it all cleaned up.

Reply:
Maybe, but I don't think it's necessary to worry about, it all goes into your tmp dir anyways.